src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -116,6 +117,7 @@ class HuiGaugeCard extends LitElement implements LovelaceCard {
           .min=${this._config.min!}
           .max=${this._config.max!}
           .value=${state}
+          .valueFormatted=${formatNumber(state, this.hass!.language)}
```

Comment:
Let's have the gauge element handle this.

Reply:
@bramkragten I'll make this change, however it requires that we add a language property to the ha gauge element.